UE4SS/src/SDKGenerator/Common.cpp

```diff
@@ -123,14 +123,14 @@ namespace RC::UEGenerator
 
     auto generate_delegate_name(FProperty* property, const File::StringType& context_name) -> File::StringType
     {
-        const std::wstring property_name = sanitize_property_name(property->GetName());
+        const auto property_name = sanitize_property_name(property->GetName());
```

Comment:
Not a huge fan of using auto in this and the following contexts.

Reply:
Why not ?

Reply:
Because at a glance you can't tell what type GetName returns.  In this context based on the name of the function it's likely some form of string, but we can't really make that assumption with "Name" given FNames.
```const StringType property_name = sanitize_property_name(property->GetName());``` would be preferable

Reply:
I would argue that you don't need to know what type it returns.
I don't particularly care myself if auto is used here or not, but I don't think the creator of this PR will respond regardless so if you want this change, then you're probably gonna have to make it yourself.